components/service/glean/metrics.yaml

```diff
@@ -95,29 +111,12 @@ glean.baseline:
       - telemetry-client-dev@mozilla.com
     expires: never
 
-  locale:
-    type: string
-    lifetime: application
-    send_in_pings:
-      - baseline
-    description: |
-      The locale of the application (e.g. "es-ES").
-    bugs:
-      - 1512938
-      - 1525540
-    data_reviews:
-      - https://bugzilla.mozilla.org/show_bug.cgi?id=1512938#c3
-    notification_emails:
-      - telemetry-client-dev@mozilla.com
-    expires: never
-
-glean.internal.metrics:
   client_id:
     type: uuid
     description:
       A UUID uniquely identifying the client.
     send_in_pings:
-      - glean_ping_info
+      - glean_client_info
```

Comment:
Please note that changing this will reset the client ids and all the other metrics. I think this should be ok right now, since nobody should be relying on it.